config/configure_notebook.py

```diff
@@ -18,8 +19,20 @@
 # COMMAND ----------
 
 sector = 'i22' # Brewing companies
-s3_bucket = 'db-industry-gtm'
-landing_zone = 'fsi/datasets/digitization/csr/files'
-landing_zone_fs = 's3://{}/{}/**/pages'.format(s3_bucket, landing_zone)
+landing_zone = '/tmp/fsi/datasets/digitization/csr/files'
 model_name = 'table_classification'
 k = 3
+landing_zone_fs = '{}/**/pages'.format(landing_zone)
+s3_bucket = 'db-industry-gtm'
```

Comment:
Remove deps if not needed

Reply:
Done in [cdc1f6b](https://github.com/databricks-industry-solutions/digitization-documents/pull/8/commits/cdc1f6b798961a99882747a7c50e1d0cd04e5dab)